doc/index.rst

```diff
@@ -42,43 +44,89 @@ game without learning a complex framework.
     <p><a href="https://vimeo.com/662273899">Quick Arcade Library Introduction Video</a></p>
    </div>
 
-The library was originally designed to be used in an educational setting, but
-it has evolved to be a powerful library for creating games. It is used in
-education, prototyping, and game jams. Hobbyists wanting to create a game
-quickly can benefit from the library's simple API.
-
+Open Source, Libre Assets
+-------------------------
 
-Documentation
--------------
+Arcade is free to use, and that :ref:`includes commercial projects <faq_commercial>`.
+We make sure licensing stays out of the way so you can of focus on really matters: creating.
```

Comment:
remove ': creating' is redundant and has a different narrative voice that the rest of the doc.